addon/components/ember-tbody/component.js

```diff
@@ -94,6 +94,14 @@ export default Component.extend({
   */
   selection: null,
 
+  /**
+    A function that will override how selection is compared to row value.
+
+    @argument selectionMatchFunction
+    @type function?
+  */
```

Comment:
This gets displayed in the docs for the `ember-tbody` component:

![Docs](https://user-images.githubusercontent.com/250934/101212641-13575300-3647-11eb-896d-8c6fbc754a5c.png)